runtime/src/accounts_db.rs

```diff
@@ -493,11 +493,15 @@ impl AccountStorageEntry {
         }
     }
 
-    fn remove_account(&self, num_bytes: usize) -> usize {
+    pub fn accounts(&self, offset: usize) -> Vec<StoredAccountMeta> {
```

Comment:
:+1: 

nits: how about omitting the `offset` and renaming to `all_accounts`? I always wonder what `0` means in the `...accounts(0)` when I read the code. Also, it seems that all of current caller are passing `0`. ;)